ext/settings.html

```diff
@@ -1581,6 +1581,47 @@ <h1>Yomichan Settings</h1>
                 <button class="icon-button"><span class="icon-button-inner"><span class="icon" data-icon="material-right-arrow"></span></span></button>
             </div>
         </div></div>
+        <div class="settings-item advanced-only">
```

Comment:
It would probably be better to use a single enum dropdown option for this instead of two toggles, something like:

**Show Anki card tags**

* Never
* Always
* Non-standard tags

Maybe add a More.../(?) link to explain what _non-standard tags_ are, if you think it's unclear.